views/js/pciCreator/ims/mathEntryInteraction/creator/tpl/propertiesForm.tpl

```diff
@@ -220,14 +220,14 @@
     <label class="panel">
         <input name="tool_lte" type="checkbox" {{#if tool_lte}}checked="checked"{{/if}}/>
         <span class="icon-checkbox"></span>
-        &le; {{__ "lower than or equal"}}
+        {{__ "lower than or equal symbol"}} {{__ "lower than or equal"}}
```

Comment:
I just though about an idea: we can use the string `≤` and `≥` as a translation key? So it will fallback to it for other language instead of changing all translations files? 

So having `{{__ "≤"}}` gives you the opportunity to update it for a language but keeps the default string.

Reply:
I just thought about the same thing actually, will do

Reply:
Just finished with an update on this. Could you please check it after` php tao/scripts/taoUpdate.php`, because I can't check it locally with my current version of tao.